### DISCUSSION TITLE
Disable update if in range (Terraform ~>)

### DISCUSSION THREAD

**Discussion by gdelbergue** [renovatebot/renovate#22947] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.122.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

Renovate does not open a MR/PR if version is in range.
For example, I defined terraform version with this code.

```hcl
required_version = "~> 1.5.0"
```

As Terraform v1.5.1 is available, Renovate opened a MR/PR with this change:

```hcl
required_version = "~> 1.5.1"
```

However, 1.5.1 is already in range of "~> 1.5.0". I would like Renovate to not open a MR/PR.

### What you tried so far.

I didn't find a solution in the doc and I didn't find topic discussing about "~>" version constraint.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by secustor** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#rangestrategy
